### DISCUSSION TITLE
Welcome to docker-nginx-php-fpm Discussions!

### DISCUSSION THREAD

**Discussion by tiredofit** [tiredofit/docker-nginx-php-fpm#17] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by opcache** [comment]
Very nice!

**Comment by blacktek** [comment]
Hello,
  is there a specific reason to not support Php5.4, while there is support for Php 5.3?

**Reply by tiredofit** [reply]
I believe I once had a 5.4 image but didn't keep it up to date. The latest ones that are kept up to date are 5.6 onwards to 8.2.

**Reply by blacktek** [reply]
no chance to recover it, right? :)

**Reply by tiredofit** [reply]
I'd have to build from scratch. I can be commissioned to build one that has the features that you need. That runtime version is 10 years old now very much so EOSL from vendor.

**Reply by blacktek** [reply]
please ignore, don't do that. I converted an old website to use php5.6. thank you!
